File: admin/lib/dto/inventory/import_inventory.dart
class ImportInventory {
  final int medicationId;
  final int quantity;
  final int price;

  ImportInventory({
    required this.medicationId,
    required this.quantity,
    required this.price,
  });

  Map<String, dynamic> toJson() {
    return {'medicationId': medicationId, 'quantity': quantity, 'price': price};
  }
}

File: be/src/main/java/com/example/petclinic/dto/request/inventory/ImportInventory.java
package com.example.petclinic.dto.request.inventory;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.experimental.FieldDefaults;

@Data
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class ImportInventory {
    int medicationId;
    int quantity;
    int price;
}

